neuronav/envs/graph_env.py

```diff
@@ -43,15 +43,15 @@ def __init__(
             self.images = utils.cifar10()
 
     def generate_graph(self, structure: GraphStructure):
-        self.rewarding_states, self.edges = structure_map[structure]()
+        self.objects, self.edges = structure_map[structure]()
+        self.rewarding_states = self.objects["rewards"] # TODO: is this redundant?
```

Comment:
I think doing similarly to `GrivEnv` would be best. Have a `self.struct_objects` that is used to reinitialize the `objects` in the case that none are passed in the reset.

Reply:
Ok I think I have fixed this